src/appstream/org.cockpit-project.cockpit-kdump.metainfo.xml.in

```diff
@@ -14,4 +14,7 @@
   <launchable type="cockpit-manifest">kdump</launchable>
   <url type="homepage">https://cockpit-project.org/</url>
   <update_contact>cockpit-devel_AT_lists.fedorahosted.org</update_contact>
+  <developer id="org.cockpit-project">
+    <name>Cockpit Project</name>
+  </developer>
```

Comment:
Note, same applies to machines

Reply:
Yes, machines/files/starter-kit is on my todo list. I wanted to wait for https://github.com/cockpit-project/cockpit-podman/pull/1874 to land, next I'll apply that to the other projects.